mypy/build.py

```diff
@@ -2052,8 +2131,13 @@ def dispatch(sources: List[BuildSource], manager: BuildManager) -> Graph:
     # just want to load in all of the cache information.
     if manager.use_fine_grained_cache():
         process_fine_grained_cache_graph(graph, manager)
+        manager.proto_deps = read_protocol_cache(manager, graph)
```

Comment:
`read_protocol_cache` can return `None` if it can't load the file. Can we handle it here? Without protocol dependencies we won't be able to produce correct results, so it looks like handling it somehow is the right thing to do. Maybe we should do a full build instead of using the cache?

Reply:
I was thinking about showing a blocking error and stopping the build. There is a TODO about this in `update.py`.